<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgMultigridPatternOptions</class>
 <widget class="QWidget" name="WdgMultigridPatternOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>243</width>
    <height>407</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabShapes">
      <attribute name="title">
       <string>Shapes</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="KisSliderSpinBox" name="sldDimensions" native="true"/>
       </item>
       <item>
        <widget class="KisSliderSpinBox" name="sldDivisions" native="true"/>
       </item>
       <item>
        <widget class="KisDoubleSliderSpinBox" name="sldOffset" native="true"/>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabLines">
      <attribute name="title">
       <string>Lines</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Line Width:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="spnLineWidth"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="gbConnector">
         <property name="title">
          <string>Connector Lines</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Type:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="cmbConnectorType"/>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spnConnectorWidth"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Width:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabColor">
      <attribute name="title">
       <string>Colors</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string comment="Line color">Lines: </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="KisColorButton" name="bnLineColor"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string comment="Connector line color">Connector:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="KisColorButton" name="bnConnectorColor">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="KisStopGradientEditor" name="wdgGradient" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QGroupBox" name="gbColors">
         <property name="title">
          <string>Color Factors</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="1" column="0" colspan="2">
           <widget class="KisDoubleSliderSpinBox" name="sldColorRatio" native="true"/>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="KisDoubleSliderSpinBox" name="sldColorIntersect" native="true"/>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="KisDoubleSliderSpinBox" name="sldColorIndex" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisStopGradientEditor</class>
   <extends>QWidget</extends>
   <header>kis_stopgradient_editor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KisDoubleSliderSpinBox</class>
   <extends>QWidget</extends>
   <header>kis_slider_spin_box.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KisColorButton</class>
   <extends>QPushButton</extends>
   <header>kis_color_button.h</header>
  </customwidget>
  <customwidget>
   <class>KisSliderSpinBox</class>
   <extends>QWidget</extends>
   <header location="global">kis_slider_spin_box.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
